crates/snforge-scarb-plugin/tests/e2e/happy_path/Scarb.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "tests2"
+version = "0.1.0"
+
+[dependencies]
+snforge_scarb_plugin = { path = "../../../" }
+snforge_std = { path = "../../../..//../snforge_std" }
```

Comment:
Is it possible to hide `snforge_scarb_plugin` under the `snforge_std` as dep?

Reply:
Yes, it will be done this way. Referenced code is just prototype test